### DISCUSSION TITLE
How do I submit a PR?

### DISCUSSION THREAD

**Discussion by robgics** [PySlurm/pyslurm#311] [General]
This isn't related to the job_info post.  I've found a bug in the job.pyx class.  I've created a branch and attempted to push it so a PR could be created but it says permission is denied.  Thanks.

**Comment by tazend** [comment]
Hi,

if you found a bug, please first [open an issue](https://github.com/PySlurm/pyslurm/issues/new) stating the exact problem you are facing.

To submit a PR, the usual way is to fork the pyslurm repo, make your changes in a branch on the fork, and contribute that back here. (When you select your branch of the fork on GitHub there is a "contribute" button, so will you be able to open a PR)

**Reply by robgics** [reply]
Ok, will do.  I forgot about the forking part.  Thanks.
